app/views/category_expenditures/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with model: [@category, @category.categories_expenditure.create] do |form| %>
+  <% if categories_expenditure.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize categories_expenditure.errors.count, "error" %> prohibited this categories_expenditure from being saved:</h2>
+
+      <ul>
+        <% categories_expenditure.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="mb-3">
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div class="mb-3">
+    <%= form.label :amount, style: "display: block" %>
+    <%= form.text_field :amount %>
+  </div>
+
+  <%= form.hidden_field :category_id, value: category_id.id  %>
+
+  <div>
+    <%= form.submit %>
```

Comment:
- [ ] Your app working fine but you should stick to the design as required font, colors, layout and space elements as in the requirements 
- Screenshot of the requirements 
<img width="522" alt="Screenshot 2022-11-17 at 13 13 06" src="https://user-images.githubusercontent.com/61790587/202443960-d4cd1cc0-74fe-44fd-bd6e-b25dec798ba5.png">

- [ ] it would be better if you add back button in the new category in order to match the requirements 
-
<img width="947" alt="Screenshot 2022-11-17 at 13 48 51" src="https://user-images.githubusercontent.com/61790587/202450774-8b944294-5f6b-4d12-97a4-4be2091b0076.png">
- Yours 
<img width="386" alt="Screenshot 2022-11-17 at 13 46 56" src="https://user-images.githubusercontent.com/61790587/202450806-eb5cff91-6809-4a14-afae-860a172ca318.png">

Reply:
I have fulfilled this requirement, but again, I honestly don't see the point. When the user saves the new category, it immediately redirects. I also have a navigation bar the same purpose.